README.md

```diff
@@ -78,7 +78,22 @@ default gems, except for
 - Gems that only appear under the `:development` group
 - All Ruby files under `test/**/*.rb`
 
-By creating a `.index.yml` file, these configurations can be overridden and tuned. Note that indexing dependent behavior, such as definition, hover, completion or workspace symbol will be impacted by the configurations placed here.
+To change this behaviour, you can pass configuration as initalization options, for example:
+
+```jsonc
+// PROJECT/.vscode/settings.json
+{
+  "rubyLsp.excludedPatterns": ["**/test/**.rb"],
```

Comment:
@vinistock is this syntax correct? I tried to test it by excluding `lib/ruby_lsp/parameter_scope.rb` then using Go To Definition on `ParameterScope`, but it was still able to locate it.

Reply:
Hmm, no, same issue.

I'm unclear on the intended syntax, since we call into `configuration.get("indexing"),` but in `lsp.code-workspace` there's no `indexing` key.

Reply:
It should be like this (and we need to fix the [workspace file](https://github.com/Shopify/ruby-lsp/blob/085e6772162a3714e6b01847178ca181f2c1e065/lsp.code-workspace#L21)):

```json
{
  "rubyLsp.indexing": {
    "excludedPatterns": ["**/test/**.rb"]
  }
}
```

Notice that in the workspace configuration, you need to apply these in `lsp.code-workspace`. I'm not sure `.vscode/settings.json` will work.